tests/test_assessment_json.py

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env pytest -vs
 """Tests for assessment JSON."""
 
-# cisagov Libraries
-import assessment.builder as assessment_builder
+# # Standard Python Libraries
```

Comment:
I'm not a fan of leaving commented-out code lying around, but if you're intending to come back to it soon then OK.

Reply:
Agreed- @bjb28 is working on fixing this test this week, so I think we will be ok here.